WcaOnRails/app/models/round.rb

```diff
@@ -66,13 +71,13 @@ def name
 
   def time_limit_to_s
     time_str = SolveTime.new(competition_event.event_id, :best, time_limit.centiseconds).clock_format
-    case self.time_limit.cumulative_round_ids.length
+    case self.time_limit.cumulativeRoundIds.length
```

Comment:
I made this change for consistency with our other serializable round attributes (such as cutoff, which currently uses snake_case: https://github.com/thewca/worldcubeassociation.org/blob/54b9f1a42b68b3600338c215c391e17935db522a/WcaOnRails/lib/cutoff.rb).

You're right though, we should definitely do camelCase inside ruby, otherwise we'll go insane. I'll attempt to make this change.